### PR TITLE
Typo in docs under Bio

### LINKED ISSUE #3042: [DOCS] Typo in docs under Bio
### Description

Under `Customising your profile` > `Bio` > "... It allow for the use..." > should be `allows`

### Screenshots

typo in Bio section

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by HarshDeep61034**
changed from allow to allows.

closes #3042 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3421"></a>

### CODE REVIEW

**Review by iamthecloverly — APPROVED** [review]

**Comment by HarshDeep61034** [comment]
Thanks @iamthecloverly

**Comment by Rishav-12** [comment]
You should also make the change in https://github.com/EddieHubCommunity/LinkFree/blob/03b907a561a70600007c6c6cc0d0836029691b65/pages/docs/index.js#L93

**Comment by HarshDeep61034** [comment]
@Rishav-12 Sure!

File: pages/docs/index.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default function DocsIndex() {
           name: "Bio",
           path: "/docs/how-to-guides/bio",
           description:
-            "The bio string in the json file is special! It allow for the use of Markdown. This will allow you to use the formatting styles like italic and bold.",
+            "The bio string in the json file is special! It allow for the use of Markdown. This will allows you to use the formatting styles like italic and bold.",
```
Comment (Aadarsh805) [inline pages/docs/index.js:93]: this is supposed to be `allow`, `allows` is wrong here

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: pages/docs/index.js (hunk 1 above)
Comment (Aadarsh805) [reply pages/docs/index.js:93]: @Rishav-12 meant the allow after the It, `It allow for the use of Markdown` -> `It allows for the use of Markdown`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by Rishav-12** [comment]
You changed the wrong allow in index.js

**Comment by HarshDeep61034** [comment]
my bad @Aadarsh805  @Rishav-12 let me fix this!

**Comment by HarshDeep61034** [comment]
Thanks, @Aadarsh805 @Rishav-12  for your feedback.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by HarshDeep61034** [comment]
Thanks @Pradumnasaraf

**Pull request merged**